itenium-socks/src/app/socks/sock-reviews.component.html

```diff
@@ -1,3 +1,12 @@
+@if ((reviews$ | async)?.length === 0){
```

Comment:
ATTENTION:
Heb je de network tab in de gaten gehouden?
Ben je hier 2x de reviews aan het ophalen?